documentation/ipi-install/ipi-install-configuration-files.adoc

```diff
@@ -25,3 +25,5 @@ endif::[]
 include::modules/ipi-install-root-device-hints.adoc[leveloffset=+1]
 
 include::modules/ipi-install-creating-the-openshift-manifests.adoc[leveloffset=+1]
+
+include::modules/ipi-install-configure-network-components-to-run-on-the-control-plane.adoc[leveloffset=+1]
```

Comment:
Is this for ALL releases or only after a specific one?

Reply:
I made it 4.8 specific given Andrew's comment.